### PR TITLE
Load Committee Members from API

### PR DESCRIPTION
**Pull request by MatthijsKok**

### CODE REVIEW

File: scripts/commission.json [hunk 1]
```diff
@@ -1,41 +1,41 @@
 [
   {
-    "function": "Chairman",
-    "name": "Beer van der Drift",
+    "function": "Chairwoman",
```
Comment (skamoen) [inline scripts/commission.json]: Chairwoman is not a thing. Please. Chairman is the universal term.

**Comment by MatthijsKok** [comment]
https://github.com/AreaFiftyLAN/lancie-api/pull/300 Needed

**Comment by skamoen** [comment]
This looks good, but let's make the whole thing dynamic in one go, including additional lancie-admin things.

**Comment by FrancisBehnen** [comment]
@MatthijsKok Can you pick this up again?

**Comment by MatthijsKok** [comment]
Unblocked because AreaFiftyLAN/lancie-api#436 is merged.

File: src/lancie-commission/lancie-commission.html [hunk 2]
```diff
@@ -68,10 +68,7 @@
       <iron-media-query query="max-width: 950px" query-matches="{{small}}"></iron-media-query>
       <iron-media-query query="max-width: 600px" query-matches="{{smalldevice}}"></iron-media-query>
 
-      <iron-ajax auto url="{{json}}"
-                 last-response="{{commission}}"
-                 handle-as="json"
-                 debounce-duration="300"></iron-ajax>
+      <lancie-ajax id="ajaxCommission" refUrl="/web/committee" handle-as="json" last-response="{{commission}}"></lancie-ajax>
```
Comment (martijnjanssen) [inline src/lancie-commission/lancie-commission.html]: This does work, however, I would change the `last-response` to an `on-response` for consistency.

File: src/lancie-commission/lancie-commission.html [hunk 3]
```diff
@@ -84,7 +81,7 @@
       </div>
       <div class$="{{_computeSmall('visual', small)}}">
         <div class="layout horizontal wrap start-justified">
-          <template is="dom-repeat" items="[[commission]]" as="commissioner">
+          <template is="dom-repeat" items="[[commission]]" as="commissioner" sort="_computeSort">
             <lancie-commission-member commissioner="{{commissioner}}" class$="{{_computeSmallDevice(smalldevice)}}"></lancie-commission-member>
```
Comment (martijnjanssen) [inline src/lancie-commission/lancie-commission.html]: I know this is not your code, but I would change this to a one-way binding. When looking into old code, I always check for things that can be improved.

**Pull request merged**